int32 label
bool is_collision
---
